src/backend/joanie/tests/core/test_api_certificate.py

```diff
@@ -10,6 +10,8 @@
 from joanie.core.factories import (
     CertificateDefinitionFactory,
     CourseProductRelationFactory,
+    EnrollmentCertificateFactory,
+    EnrollmentFactory,
```

Comment:
maybe it's time to import factories :sweat_smile:

Reply:
Indeed ^^